db-drda/src/main/java/com/splicemachine/db/impl/drda/DRDAConnThread.java

```diff
@@ -8725,6 +8752,7 @@ private void handleException(Exception e)
             if (e instanceof DRDAProtocolException) {
                 // protocol error - write error message
                 sendProtocolException((DRDAProtocolException) e);
+                server.consoleExceptionPrintTrace(e);
```

Comment:
`LOG.debug(message, error)`

Reply:
I would make it part of this [JIRA](https://splicemachine.atlassian.net/browse/DB-10893).